examples/_includes/macros.nj

```diff
@@ -0,0 +1,16 @@
+{% macro description(x) %}
+<text style="description">{{ x }}</text>
+{% endmacro %}
+
+{% macro button(text, buttonProps={}) %}
+<view
+  {% for attr, val in buttonProps %}
```

Comment:
minor/naming: I would suggest to use `attributes` instead of buttonProps, to have a common way to specify XML attributes for every macro and to better reflect how this parameter is used.